src/rebar_api.erl

```diff
@@ -5,7 +5,11 @@
 -include_lib("providers/include/providers.hrl").
 -export([abort/0, abort/2,
          console/2,
-         debug/2, info/2, warn/2, error/2]).
+         debug/2, info/2, warn/2, error/2,
+         expand_env_variable/3,
+         get_arch/0,
+         wordsize/0]).
```

Comment:
For port_compiler I have need of `rebar_utils:sh/2` to be re-exported from this module.